File: src/main/java/com/example/ssauc/user/recommendation/dto/RecommendationDto.java
package com.example.ssauc.user.recommendation.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class RecommendationDto {
   Long productId;
   String name;
   String description;
   Long price;
   String location;
   String imageUrl;
}

File: src/main/java/com/example/ssauc/user/recommendation/dto/RecommendationDto.kt
package com.example.ssauc.user.recommendation.dto

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
class RecommendationDto {
    var productId: Long? = null
    var name: String? = null
    var description: String? = null
    var price: Long? = null
    var location: String? = null
    var imageUrl: String? = null
}
